Add missing `CHANGELOG.md` entry for symbol uploading script fix (#546)
getsentry/sentry-unreal: CHANGELOG.md

### Fixes

- The SDK no longer prints symbol uploading related warnings when disabled ([#528](https://github.com/getsentry/sentry-unreal/pull/528))

> - Fix problem with parsing config file during symbol uploading ([#541](https://github.com/getsentry/sentry-unreal/pull/541))

Reviewer: Fixed an issue when parsing the config file during symbol upload

> - Fix problem with parsing config file during symbol uploading ([#541](https://github.com/getsentry/sentry-unreal/pull/541))

Reviewer: ```suggestion
- Fixed an issue when parsing the config file during symbol upload ([#541](https://github.com/getsentry/sentry-unreal/pull/541))
```
